code/__DEFINES/gamemode.dm

```diff
@@ -48,7 +48,7 @@
 #define ROLE_MUTINEER "mutineer"
 #define ROLE_COMMANDO "commando"
 #define ROLE_DEATHSQUAD "deathsquad"
-#define ROLE_ACTOR "actor"
```

Comment:
Uhhhhh, Tori? Are we removing Actor without making it an alt-title for club workers?

Reply:
Regression that I missed, will undo.